### PR TITLE
Add payouts

### PR DESCRIPTION
**Pull request by olssonm**
This PR enables [payouts](https://developer.swish.nu/api/payouts/v1) via the Swish API (see #7).

Might require a major semver update depending on how the certificates and signing processes are handled.

### CODE REVIEW

**Comment by vinkla** [comment]
Great work Marcus 🔥 

I highly recommend releasing this under a [branch alias](https://getcomposer.org/doc/articles/aliases.md) if possible, then we can test it out and fix any quirks before releasing a new stable version.

**Comment by olssonm** [comment]
@vinkla Yep, will do – and/or possible publish a pre-release. Still a few quirks to iron out, will ping when something is available to test.

**Comment by olssonm** [comment]
@vinkla You should now be able to require the development-branch with:

``` php
"repositories": [
        {
            "type": "vcs",
            "url": "git@github.com:olssonm/swish-php.git"
        }
    ],
```

And running `composer require olssonm/swish-php:dev-dev/payouts`

**Comment by vinkla** [comment]
Thanks! I'll test this as soon as possible.

File: src/config.php [hunk 1]
```diff
@@ -5,6 +5,8 @@
         'client' => env('SWISH_CLIENT_CERTIFICATE_PATH'),
         'password' => env('SWISH_CLIENT_CERTIFICATE_PASSWORD'),
         'root' => env('SWISH_ROOT_CERTIFICATE_PATH', true),
+        'signing' => env('SWISH_SIGNING_CERTIFICATE_PATH', null), // Optional, used for payouts
```
Comment (vinkla) [inline src/config.php:8]: I suggest wrapping these in a `storage_path` helper function or providing an example of how to add these in the `.env` file.

Comment (olssonm) [reply src/config.php:8]: @vinkla Yes, should be clarified that it's the full/absolute path of the file that is needed. Using `storage_path` is not optimal as the certificates might be stores outside of Laravel and/or the .env-file might be used by multiple apps in the same environment with different storage paths.

File: src/config.php [hunk 2]
```diff
@@ -5,6 +5,8 @@
...
+        'signing_password' => env('SWISH_SIGNING_CERTIFICATE_PASSWORD', null), // Optional, used for payouts
     ],
     'endpoint' => \Olssonm\Swish\Client::PRODUCTION_ENDPOINT,
```
Comment (vinkla) [inline src/config.php:11]: @olssonm what do you think about adding this as a environment variable as well?

```suggestion
    'endpoint' => env('SWISH_URL', \Olssonm\Swish\Client::PRODUCTION_ENDPOINT),
```

Comment (olssonm) [reply src/config.php:11]: @vinkla Good call, definitely! Will add it in a separate PR soon, as Swish now also provides a staging environment that will be added as a constant.

**Review by vinkla — APPROVED** [review]
Great work! The test API appears to be functioning as expected 🙌

**Pull request merged**